Makefile.am

```diff
@@ -229,6 +233,15 @@ EXTRA_DIST += \
 	nmsg/base/pkt.c \
 	nmsg/base/xml.c
 
+FLT_LIBTOOL_FLAGS = -module -avoid-version -shared -export-symbols-regex "^nmsg_fltmod_plugin_export$$"
```

Comment:
Yes.  Expanding a bit on the existing section in README.md for how to build msgmod and fltmod plugins that references the appropriate headers with their documentation, makes note of installation paths, and links to the Autotools Mythbuster would probably be enough.